plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.properties

```diff
@@ -3,4 +3,4 @@ providerName=Google, Inc.
 
 deployStandardCommandDescription=Deploy the project to App Engine Standard
 deployStandardCommandName=Deploy to App Engine Standard
```

Comment:
For future reference (thanks @briandealwis ):
 - command name is used when a menu label is not provided or the handler does not implement the `org.eclipse.ui.commands.IElementUpdater` interface, it also appears in the `Quick Access (Ctrl+3)` view
 - command description is usually a tooltip, but also appears in the `Quick Access (Ctrl+3)` view
 - menu extension label (the value of `deployStandardMenuLabel` property in this case) is shown in the menu this command is added to.

I'll update the command description.